src/elements/Tabs/TabsContainer.tsx

```diff
@@ -19,21 +23,77 @@ interface Indicator {
   Component: React.FC<TabItemProps<string>>
 }
 
+interface PillTabItemProps extends TabItemProps<string> {
+  onTabPress?: (tabName: string) => void
+  focusedTab: { value: string }
+}
+
+const PillTabItem: React.FC<PillTabItemProps> = (props) => {
+  const [focusedTab, setFocusedTab] = useState(props.focusedTab.value)
+
+  // We want to debounce the focusedTab value to avoid showing two pills at once
+  const debouncedFocusedTab = debounce(setFocusedTab, 30)
+
+  useAnimatedReaction(
+    () => props.focusedTab.value,
+    (value) => {
+      runOnJS(debouncedFocusedTab)(value)
+    }
+  )
+
+  return (
+    <Pill
+      selected={props.name === focusedTab}
+      onPress={() => {
+        props.onTabPress?.(props.name)
+      }}
+      key={props.name}
+      variant="link"
+    >
+      {props.name}
+    </Pill>
+  )
+}
+
+interface DefaultTabItemProps extends TabItemProps<string> {
+  indicators: Indicator[]
+  onPress: (name: string) => void
+}
+
+const DefaultTabItem: React.FC<DefaultTabItemProps> = (props) => {
+  const Indicator = props.indicators.find((indicator) => {
+    return indicator.tabName === props.name
+  })
+
+  return (
+    <Box flex={1}>
+      <Flex position="absolute" width="100%">
+        {!!Indicator?.Component && <Indicator.Component {...props} />}
+      </Flex>
+      <MaterialTabItem pressOpacity={DEFAULT_ACTIVE_OPACITY} {...props} onPress={props.onPress} />
```

Comment:
Good to use DEFAULT_ACTIVE_OPACITY 👍 

```suggestion
      <MaterialTabItem pressOpacity={DEFAULT_ACTIVE_OPACITY} {...props} />
```